src/NexusMods.Telemetry/ExceptionData.cs

```diff
@@ -15,13 +15,50 @@ public static List<ExceptionData> Create(Exception exception, List<ExceptionData
         }
         else
         {
-            result.Add(new ExceptionData(
-                Type: exception.GetType().ToString(),
-                Message: RedactUtils.Redact(exception.Message),
-                StackTrace: exception.StackTrace
-            ));
+            result.Add(From(exception));
         }
 
         return result;
     }
+
+    private static ExceptionData From(Exception exception)
```

Comment:
Want to add a comment explaining what From is meant to do and why for future reference?